modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -83,21 +84,53 @@ spec:
           name: cloud-credentials
         default: true
         objectStorage:
-          bucket: <bucket_name>
-          prefix: <bucket_prefix>
+          bucket: my-bucket
+          prefix: velero
         provider: aws
   configuration:
     restic:
-      enable: <true_or_false>
+      enable: false
     velero:
+       itemOperationSyncFrequency: "10s"
       defaultPlugins:
         - openshift
         - aws
         - csi
+        - vsm
+       resourceTimeout: 10m <1>
+  features:
+    dataMover:
+      credentialName: restic-secret
+      enable: true
+      maxConcurrentBackupVolumes: "3" <2>
+      maxConcurrentRestoreVolumes: "3" <3>
+      pruneInterval: "14"  <<<<<<<--------------------  pruneInterval <4>
+      volumeOptions:  <5>         <------- volumeOptions
```

Comment:
same, why the arrows?  Can we not call out new parameters in a comment w/ context?